arte/photometry/thermal_flux.py

```diff
@@ -2,6 +2,7 @@
 import astropy.units as u
 from synphot.spectrum import SourceSpectrum
 from synphot.models import BlackBody1D
+from synphot import units
 
     
 class ThermalFluxOnWFS():
```

Comment:
1) I'm not totally convinced of the solid angle computation: is it based on some specific geometric assumption on lenslet array and cold stops in the camera? Wouldn't be better to have the class parametrized on the input F# and possibly adding an helper function that computes the input F# accordingly to the SH WFS parameters. This has also the advantage that the class is more generic: actually now it should be named ThermalFluxOnSHWFS
2) Is lenslet size the size of the individual lenslet or the diameter/radius of the lenslet array?
3) Add a test of the class, even a stupid one, that works also as example